test/controllers/SecretControllerSpecGetAPITest.scala

```diff
@@ -0,0 +1,27 @@
+package controllers
+
+import org.scalatestplus.play._
+import org.scalatestplus.play.guice._
+import play.api.test.Helpers._
+import play.api.test._
+import stubs.StubSecretService
+
+class SecretControllerSpecGetAPITest extends PlaySpec with GuiceOneAppPerTest {
+
+  def validGetSecreteControllerCallReturnsPassword() = {
```

Comment:
`shouldReturnPasswordForASecrete` always use agreement like statements for the test method names.

Practice it more

Reply:
sure